rotkehlchen/inquirer.py

```diff
@@ -369,10 +405,26 @@ def find_usd_price(
             except RemoteError:
                 pass  # continue, a price can be found by one of the oracles (CC for example)
 
+        # Try and check if it an ethereum token with protocol or underlying tokens
+        is_known_protocol = False
+        underlying_tokens = None
+        try:
+            token = EthereumToken.from_asset(asset)
+            if token is not None:
+                if token.protocol:
+                    KnownProtocolsAssets(token.protocol)
+                    is_known_protocol = True
+                underlying_tokens = GlobalDBHandler().get_ethereum_token(  # type: ignore
+                    token.ethereum_address,
+                ).underlying_tokens
+        except ValueError:
```

Comment:
again the value error. Why? -- Could it be due to the token.protocol not being part of the enum. If so please for the love of god don't do that. Never use exceptions for control handling: https://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why

What you should do as mentioned elsewhere is just check if the protocol is part of the list/tuple/set of known protocols.